go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/shirou/gopsutil/v3 v3.20.11
 	github.com/syncthing/notify v0.0.0-20201210100135-17de26665ddc
 	github.com/syndtr/goleveldb v1.0.1-0.20200815071216-d9e9293bd0f7
+	github.com/thejerf/suture v4.0.0+incompatible
```

Comment:
That shouldn't be here. Also in go.sum.

Reply:
My ide keeps adding it, not sure why it can't deal with the suture upgrade. Doesn't like the /v4 import. I think it should be suture, and go.mod just makes sure that resolves to v4m

Reply:
Since modules the import path must include the major version suffix (e.g. it's possible to use both v3 and v4 of the same library in a module). Maybe a leftover uncommitted file that still imports "github.com/thejerf/suture"?